frame/nfts/src/lib.rs

```diff
@@ -101,6 +101,16 @@ pub mod pallet {
 			+ IsType<<Self as frame_system::Config>::RuntimeEvent>;
 
 		/// Identifier for the collection of item.
+		///
+		/// SAFETY: The functions in the `Incrementable` trait are fallible.
+		/// If the functions of the implementation both return `None`,
+		/// the automatic CollectionId generation should not be used.
+		/// So the `create` and `force_create` extrinsics
```

Comment:
line wrapping seems off.

Reply:
vscode extension marketplace > ReWrap

Reply:
@aaronbassett showerthought: this might be something nice to cover in one of your live streams

Reply:
>  "rewrap.wrappingColumn": 95,

:o I have mine on 100